src/anemoi/utils/checkpoints.py

```diff
@@ -213,6 +215,15 @@ def save_metadata(
             )
             zipf.writestr(entry["path"], value.tobytes())
 
+        for name, entry in metadata["supporting_arrays_paths"].items():
+            if isinstance(entry, dict):
+                for subname, subentry in entry.items():
+                    subvalue = supporting_arrays[name][subname]
+                    save_supporting_array(f"{name}/{subname}", subentry, subvalue)
```

Comment:
I am not sure if we want to use nested dicts or keys with `{name}/{subname}` or `{name}.{subname}`.